frame/evm/src/runner/stack.rs

```diff
@@ -1170,9 +1184,35 @@ where
 			code.len(),
 			address
 		);
+
 		Pallet::<T>::create_account(address, code, caller)
 	}
 
+	fn set_delegation(
+		&mut self,
+		authority: H160,
+		delegation: evm::delegation::Delegation,
+	) -> Result<(), ExitError> {
+		log::debug!(
+			target: "evm",
+			"Inserting delegation (23 bytes) at {:?}",
+			delegation.address()
+		);
+
+		Pallet::<T>::create_account(authority, delegation.to_bytes(), None)
```

Comment:
This will still call `frame_system::Pallet::<T>::inc_sufficients(who)`, I do not think we want it, since EOA already incremented `sufficients`. The problem is that `sufficients` will be incremented everytime a new delegation is provided.